scripts/cpplinter.sh

```diff
@@ -1,8 +1,42 @@
 #!/bin/bash
 
+# Copyright Intel Corp. 2019 All Rights Reserved.
 # Copyright IBM Corp. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
 
-# TODO Implement me
+SCRIPT=`realpath $0`
+SCRIPTDIR=`dirname $SCRIPT`
 
+FILES_TO_PARSE=`find $SCRIPTDIR/.. -iname *.cpp -o -iname *.h -o -iname *.c`
+
+RET=0
+
+for FILE in $FILES_TO_PARSE; do
+
+    #skip file checking for certain folders
+    echo $FILE | grep 'ecc_enclave/_build'  > /dev/null && continue
```

Comment:
putting this exclusion list as a var and then loop through it would seem a bit nicer, though, adding something in the pattern here is admittedly not much more complicated ..

Reply:
fixed -- same complexity though more concise